### DISCUSSION TITLE
StepZen Stream on Friday at 12:00pm PT

### DISCUSSION THREAD

**Discussion by ajcwebdev** [stepzen-dev/stepzen-nuxt#17] [Announcements]
Hello 👋! We'd like to highlight the work everyone has contributed to Hacktoberfest so far. On Friday at 12:00pm PT, Lucia and I will be on the StepZen Stream and any Hacktoberfest participants are welcome to join us to share their experience.

@codiini @crweiner @annetawamono @emmebravo @virtuoushub

If you would like to join us please respond to this discussion thread.

**Comment by annetawamono** [comment]
Hey @ajcwebdev , how long will the stream be? I'd like to join but I have to be somewhere at 1pm PT.

**Reply by ajcwebdev** [reply]
That's fine, we'll be finished by 1pm and if you need to leave earlier that's fine as well.

**Comment by crweiner** [comment]
Thank you for the invite, and sadly I won't be able to attend. Have fun though!! 😄

**Comment by ajcwebdev** [comment]
For @annetawamono and anyone else who decides to join us tomorrow:

I will post a StreamYard link here about 5-10 minutes before the start of the stream tomorrow (stream starts at 12:00pm PT). You'll click the link to join the studio and then I will add you to the stage. If you want to join a little before the start of the stream we can do an audio and video check.

**Comment by ajcwebdev** [comment]
Here is the StreamYard link for anyone who would like to join us at the top of the hour:

https://streamyard.com/w67c84e2xz
